view/common/block-layout/browse-preview.phtml

```diff
@@ -31,12 +31,16 @@ foreach ($this->resources as $resource):
     $body = $bodyTerm ? $resource->value($bodyTerm) : $resource->displayDescription();
 ?>
     <li class="<?php echo $this->resourceType; ?> resource">
-        <?php if ($thumbnail = $this->thumbnail($resource, 'medium')): ?>
-        <?php echo $resource->linkRaw($thumbnail, null, ['class' => 'thumbnail']); ?>
```

Comment:
Ah, the return of the list layout made me notice the omission of the thumbnail class.